Added baked potato recipe (#570)
LukeSmithxyz/based.cooking: src/baked-potatoes.md

# Baked Potatoes

- ⏲️ Prep time: 10 minutes
- 🍳 Cook time: 20-25 minutes
- 🍽️ Servings: 4-6 people

## Ingredients

- 6-7 Yukon potatoes (you can use any potatoes you like)
- 1/3 cup of any vegetable oil

## Directions
1. Preheat the oven to 425°F/220°C
2. Wash potatoes and cut into into 5mm slices
3. Put the sliced potatoes into a bowl
4. Add salt, pepper and any other seasoning you like
5. Add 1/3 cup of a vegetable oil
6. Add salt, pepper and any other seasonings you like
7. Mix everything and put it on a baking sheet (optionally lined with baking paper to make cleanup easier) for 20 to 25 minutes.

> I preffer to use baking paper, it makes everything cleaner and easier. Put your potatoes on the baking pan and put it on the oven for 20-25 minutes.
> 
> ## Contribution

Reviewer: ```suggestion
## Contribution
```

Can probably get rid of this since it is in step 7 now.

# Baked Potatoes

- ⏲️ Prep time: 10 minutes
- 🍳 Cook time: 20-25 minutes
- 🍽️ Servings: 4-6 people

## Ingredients

> - 6-7 Yukon pototoes (you can use any potatoes you like)

Reviewer: ```suggestion
- 6-7 Yukon potatoes (you can use any potatoes you like)
```

> - 1/3 cup of any vegitable oil

Reviewer: ```suggestion
- 1/3 cup of any vegetable oil
```

> - 1 teaspoon salt
> - Black pepper
> - Any seasoning you like

Reviewer: ```suggestion
```

Remove salt/pepper and seasonings 'to taste' from ingredients. They can be mentioned below in the directions.

## Directions

> 1. Preheat the oven to 425F/220C

Reviewer: ```suggestion
1. Preheat the oven to 425°F/220°C
```

> 2. Wash potatoes and slice into 5mm slices

Reviewer: ```suggestion
2. Wash potatoes and cut into into 5mm slices
```

> 3. Put the sliced poptatoes into a bowl

Reviewer: ```suggestion
3. Put the sliced potatoes into a bowl
```

4. Add salt, pepper and any other seasoning you like

> 5. Add 1/3 cup of a vegitable oil

Reviewer: ```suggestion
5. Add 1/3 cup of a vegetable oil
```

> 6. Mix everything and put it on the baking pan

Reviewer: ```suggestion
6. Mix everything and put it on a baking sheet (optionally lined with baking paper to make cleanup easier) for 20 to 25 minutes.
```

> I preffer to use baking paper, it makes everything cleaner and easier. Put your potatoes on the
> baking pan and put it on the oven for 20-25 minutes.
> 

Reviewer: ```suggestion
```
